src/c/admin-radio-action.js

```diff
@@ -21,11 +21,11 @@ window.c.AdminRadioAction = (function(m, h, c, _){
 
       setFilter[updateKey] = 'eq';
       var setVM = m.postgrest.filtersVM(setFilter);
-      setVM[updateKey](item[updateKey]);
+      setVM[updateKey](contribution[updateKey]);
```

Comment:
Maybe we could just pass updateKeyValue as an argument, instead of contribution.